mkdocs_jupyter/convert.py

```diff
@@ -84,7 +94,14 @@ def nb2md(nb_path):
     # print(exporter.template_paths)
     # End block
 
-    body, resources = exporter.from_filename(nb_path)
+    _, extension = os.path.splitext(nb_path)
+
+    if HAS_JUPYTEXT and extension == ".py":
```

Comment:
we need to check the extension here as well otherwise it will run for both "py" & "ipynb" files